pkg/db/client.go

```diff
@@ -146,6 +169,54 @@ func (c *ClientSqlx) Exec(ctx context.Context, query string, args ...interface{}
 	return c.db.ExecContext(ctx, query, args...)
 }
 
+func (c *ClientSqlx) ExecMultiInTx(ctx context.Context, sqlers ...Sqler) (results []sql.Result, err error) {
+	var tx *sql.Tx
+	var res sql.Result
+	var buildErr error
+	var qry string
+	var queries []string
+	var args []interface{}
+	var argss [][]interface{}
+
+	for i, sqler := range sqlers {
+		if qry, args, buildErr = sqler.ToSql(); buildErr != nil {
```

Comment:
tbh i don't see the point in predefining qry, args and buildErr as vars as they are only used within the the scope of this for loop and not outside aswell.